reference-architecture/azure-ansible/azuredeploy.json

```diff
@@ -286,23 +286,23 @@
       }
     },
     "metrics" : {
-      "type": "bool",
-      "defaultValue": true,
-      "metadata" :{
+      "type" : "bool",
+      "defaultValue" : true,
+      "metadata" : {
         "description" : "Enable OCP metrics"
       }
     },
     "logging" : {
-      "type": "bool",
-      "defaultValue": true,
-      "metadata" :{
+      "type" : "bool",
```

Comment:
can you check the spaces here vs the previous code?

It looks like a lot of the "changes" aren't really changes